lib/src/widgets/player.dart

```diff
@@ -15,15 +19,35 @@ class BoardPlayer extends StatelessWidget {
     required this.player,
     this.active,
     this.clock,
+    this.diff,
     super.key,
   });
 
   final Player player;
   final Duration? clock;
   final bool? active;
+  final MaterialDiffSide? diff;
 
   @override
   Widget build(BuildContext context) {
+    final IList<Role> roleList = {
```

Comment:
that should be a `const` defined outside. But you can actually use `Role.values` instead.